faucet.go

```diff
@@ -880,23 +808,31 @@ func (l *lightningFaucet) CloseAllChannels() error {
 func (l *lightningFaucet) generateInvoice(homeTemplate *template.Template,
 	homeState *homePageContext, w http.ResponseWriter, r *http.Request) {
 
+	// Verify IP before open a channel
+	hostIP, err := getRealIP(r, l.cfg.UseRealIP)
```

Comment:
Previously the code checked if the feature was disabled before checking the client timelimit, I think this should be maintained. We should also parse the `amt` and `description` variables before rendering the homeTemplate, so that the users values are repopulated in the form.